packages/graphql/src/entities/App.ts

```diff
@@ -18,6 +18,16 @@ export class App {
     return true
   }
 
+  @nxs.field.boolean({
```

Comment:
I think this should be required - eg `true` or `false` (cannot be `undefined` - we are in global mode, or we are not).

You can add `nonNull`:

```ts
@nxs.field.nonNull.boolean
```